rsts/concepts/flyte_console.rst

```diff
@@ -0,0 +1,202 @@
+.. _ui:
+
+How to Use Flyte UI
+===================
+
+Flyte UI is a web-based user interface for Flyte. It helps interact with Flyte objects and builds DAGs out of your workflows.
+
+With Flyte UI, you can:
```

Comment:
we can say you can also do - look at registered versions of your workflows and their static graphs
launch tasks - talk about that

Reply:
lets add those and we can add the gifs later. But I like this - thank you

Reply:
I have added _launch task_. _look at registered versions of your workflows and their static graphs_ is already covered in the "View Versioned Executions" section.